label_studio_sdk/project.py

```diff
@@ -1361,3 +1384,126 @@ def connect_azure_export_storage(
         }
         response = self.make_request('POST', '/api/storages/export/azure', json=payload)
         return response.json()
+
+    def assign_by_sampling(
```

Comment:
I feel like this method should be "_internal"

Reply:
Fixed